### PR TITLE
FUSETOOLS2-2037: Migrate VSCode Camel Extensions Pack extension readme file to new format

### PR DESCRIPTION
**Pull request by djelinek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,151 @@
...
+</h1>
+
+<p align="center">
+  <a href="https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack"><img src="https://img.shields.io/visual-studio-marketplace/v/redhat.apache-camel-extension-pack?style=for-the-badge" alt="Marketplace Version"/></a>
+  <a href="https://github.com/camel-tooling/vscode-camel-extension-pack/actions/workflows/main.yml"><img src="https://img.shields.io/github/actions/workflow/status/camel-tooling/vscode-camel-extension-pack/main.yml?style=for-the-badge" alt="Main CI"></a>
+  <a href="https://github.com/camel-tooling/vscode-camel-extension-pack/blob/main/LICENSE"><img src="https://img.shields.io/github/license/camel-tooling/vscode-camel-extension-pack?color=blue&style=for-the-badge" alt="License"/></a>
+  <a href="https://camel.zulipchat.com/#narrow/stream/258729-camel-tooling"><img src="https://img.shields.io/badge/zulip-join_chat-brightgreen?color=yellow&style=for-the-badge" alt="Zulip"/></a>
+</p><br/>
+
+<h2 align="center">Camel Extensions Pack for Visual Studio Code</h2>
```
Comment (apupier) [inline README.md:14]: for legal reasons we cannot mention it this way. Given that we are not an official Apache project, it must be "Something for Camel", and never "Camel XXX"

File: README.md [hunk 2]
```diff
@@ -1,20 +1,151 @@
...
+
+<p align="center">
+This is the extensions pack that brings common developer experience for <a href="http://camel.apache.org/">Apache Camel</a> in <a href="https://code.visualstudio.com/">Visual Studio Code</a>.
```
Comment (apupier) [inline README.md:17]:
```suggestion
This is the extension pack that brings common developer experience for <a href="http://camel.apache.org/">Apache Camel</a> in <a href="https://code.visualstudio.com/">Visual Studio Code</a>.
```

File: README.md [hunk 3]
```diff
@@ -1,20 +1,151 @@
...
-* [AtlasMap Data Transformation editor](https://marketplace.visualstudio.com/items?itemName=redhat.atlasmap-viewer). It provides a graphical editor for Data Transformation based on the camel-atlasmap component.
-* [Tools for Microprofile](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-microprofile). It provides support for the development of MicroProfile®-based applications.
-* [Quarkus tools for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-quarkus). It provides Quarkus and MicroProfile development by extending Visual Studio Code extension for MicroProfile with Quarkus features.
-* *Deprecated* [Project initializer by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.project-initializer). It allows to create Apache Camel or Red Hat Fuse projects by selecting a mission.
-![Create project](./images/CreateProjectFromTopLevelCommand.gif)
+<h1 align="center">
+  <img width="125" height="125" src="https://raw.githubusercontent.com/camel-tooling/vscode-camel-extension-pack/main/icons/icon128.png">
+  <br>
+  Extensions Pack for Apache Camel
```
Comment (apupier) [inline README.md:4]:
```suggestion
  Extension Pack for Apache Camel
```

https://code.visualstudio.com/blogs/2017/03/07/extension-pack-roundup

File: README.md (hunk 1 above)
Comment (djelinek) [reply README.md:14]: I will probably remove this `h2` heading because I do not have idea what to mention here to not duplicate h1

**Pull request merged**